vrk-kpa/suomidigi
# SUOPA-866 Navigation v3

To test: 
- `make fresh`
- Open the links in https://jira.dvv.fi/browse/SUOPA-866
- Check that the desktop menu has "More" button and the navigation links move underneath the more button when the width of the browser is changed - and vice-versa
- Check that the sidebar theme is similar to proto
- Check that the mobile navigation theme is similar to proto
